website/content/docs/agent-and-proxy/proxy/caching/static-secret-caching.mdx

```diff
@@ -0,0 +1,140 @@
+---
+layout: docs
+page_title: Vault Proxy Static Secret Caching
+description: |-
+  Vault Proxy's Static Secret Caching functionality allows you to cache KVv1 and KVv2 secrets for calling clients.
+  The secrets will be automatically updated by Proxy, minimizing requests made to Vault, and offering resiliency.
+---
+
+# Vault Proxy Static Secret Caching
+
+Vault Proxy's Static Secret Caching functionality allows you to cache KVv1 and KVv2 secrets for calling clients.
+The secrets will be automatically updated by Proxy, minimizing requests made to Vault, and offering resiliency.
+
+## Required permissions
+
+To enable static secret caching, [auto-auth](/vault/docs/agent-and-proxy/autoauth) must also be enabled.
+The auto-auth token must also have permissions to subscribe to KV event updates via Vault's [event system](/vault/docs/concepts/events).
+Proxy uses the auto-auth token to connect to Vault's event system, and uses events to know when to update its cache
+with updated secrets.
+
+An example policy snippet that would grant access to KVv1 and KVv2 events is given below:
```

Comment:
I suggest saying "static secrets (KVv1 and KVv2)" just to be clear that those secrets engines are static secrets

Reply:
Good call! Updating :)